Satori/Event.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 The Satori Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Xml;
+
+namespace Satori
+{
+    /// <summary>
+    /// An event to be published to the server.
+    /// </summary>
+    public class Event
```

Comment:
Do we need this event class as a wrapper around the input arguments to `SendEventAsync`?

Reply:
No but we need it for the batch send version, `SendEventsAsync`.